docs/ABOUT.md

```diff
@@ -1,35 +1,7 @@
+[Java](https://go.java/index.html) is among the most popular available programming languages, thanks to its versatility and compatibility. It is widely used for software development, mobile applications, and large systems development.
```

Comment:
Maybe `and developing larger systems`. I also don't think there should be a comma before the `and`. But I'm no expert on comma rules, so feel free to correct me if I'm wrong :)

Reply:
Also, as part of the Exercism documentation guidelines, each sentence should start on a new line :)

Reply:
I fixed the English mistakes as you suggested. Sorry about that.

I didn't find the guideline mentioning I should start each sentence in on a new line :(
Should it be here? https://github.com/exercism/docs/blob/master/language-tracks/documentation/introductory-copy.md

I adapted my file now by following the other pages, so I hope it is in the correct format now :)

Reply:
@FridaTveit the last comma before the and, is usually known as a serial or oxford comma. There's quite a bit of controversy around it. Grammarly has a short blog post about it [here](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/)

Reply:
@RhuanKrum I remember there was a long discussion about it in the `docs` repo I think. But saying that, I can't find it now! So it's probably not very important :)

Thanks @Smarticles101 :)